evm/v0.5/src/helpers.ts

```diff
@@ -490,13 +545,24 @@ export function requestDataBytes(
   ])
 }
 
+export function depositFundsBytes(
```

Comment:
We can probably replace this function with a call to `coordinator.functions.depositFunds.encode()`